spec/03-plugins/37-opentelemetry/05-otelcol_spec.lua

```diff
@@ -37,7 +35,7 @@ for _, strategy in helpers.each_strategy() do
 
       bp.plugins:insert({
         name = "opentelemetry",
-        config = table_merge({
+        config = kong.table.merge({
```

Comment:
I'd perfer to use `utils.table_merge` instead of `kong.table.merge`, fewer Lua table lookup operations perhaps.